test/api_test.js

```diff
@@ -154,9 +149,11 @@ describe("KintoClient", () => {
         batch: false,
       };
 
-      expect(api.bucket("foo", options)).to.have
-        .property("options")
-        .eql(options);
+      const bucket = api.bucket("foo", options);
+      expect(bucket).property("_safe", options.safe);
+      expect(bucket).property("_retry", options.retry);
+      expect(bucket).property("_headers").eql(options.headers);
+      expect(bucket).property("_isBatch", options.batch);
```

Comment:
I don't think it would be useful to expose them publicly. The tests are trying to make sure that the fields are "recognized"; this seems like a simple way to do that, even if it involves poking around in the internals of the class. (Otherwise I don't see any way to write the test at all.)